modin/core/execution/unidist/common/utils.py

```diff
@@ -34,6 +32,7 @@ def initialize_unidist():
         )
 
     if not unidist.is_initialized():
+        unidist_cfg.CpuCount.put(modin_cfg.CpuCount.get())
```

Comment:
if you're doing this (and if unidist supports dynamic changing of cpus) you should make a callback on `modin_cfg.CpuCount` to put respectful change to `unidist_cfg`

could be done in a separate PR, just letting you know that the config is expected (ideally) to be treated as a dynamic thing and that its clients should respond to values being changed if that is useful for the end user

Reply:
we can do that in a separate PR if you like, though it feels like a one-line change to me, something like:
```python
modin_cfg.CpuCount.subscribe(lambda count: unidist_cfg.CpuCount.put(count.get()))
```